app/Item.php

```diff
@@ -123,6 +122,7 @@ class Item extends Model implements IndexableModel, TranslatableContract
 
     protected $casts = [
         'colors' => 'json',
+        'frontends' => 'json',
```

Comment:
Would this make a meaningful difference?

https://laravel.com/docs/10.x/eloquent-mutators#array-and-json-casting

```suggestion
        'frontends' => 'array',
```